### ISSUE TITLE
[bitnami/openresty]: 1.25.3-2 doesn't start with nginx.conf as volume

### ISSUE THREAD

**Issue by ilpianista** [bitnami/containers#71315]
### Name and Version

bitnami/openresty:1.25.3-2

### What architecture are you using?

None

### What steps will reproduce the bug?

At the moment of writing, starting openresty image using a `nginx.conf` doesn't work anymore:

```
$ docker run -it -v /tmp/nginx.conf:/opt/bitnami/openresty/nginx/conf/nginx.conf bitnami/openresty:1.25
```

### What is the expected behavior?

It should start nginx.

### What do you see instead?

```
openresty 09:54:39.66 INFO  ==> 
openresty 09:54:39.66 INFO  ==> Welcome to the Bitnami openresty container
openresty 09:54:39.66 INFO  ==> Subscribe to project updates by watching https://github.com/bitnami/containers
openresty 09:54:39.67 INFO  ==> Submit issues and feature requests at https://github.com/bitnami/containers/issues
openresty 09:54:39.67 INFO  ==> Upgrade to Tanzu Application Catalog for production environments to access custom-configured and pre-packaged software components. Gain enhanced features, including Software Bill of Materials (SBOM), CVE scan result reports, and VEX documents. To learn more, visit https://bitnami.com/enterprise
openresty 09:54:39.67 INFO  ==> 
openresty 09:54:39.67 INFO  ==> ** Starting OpenResty setup **
openresty 09:54:39.69 INFO  ==> Validating settings in OPENRESTY_* env vars
openresty 09:54:39.69 WARN  ==> The OpenResty configuration file '/opt/bitnami/openresty/nginx/conf/nginx.conf' is not writable by current user. Configurations based on environment variables will not be applied.
openresty 09:54:39.69 INFO  ==> Loading custom scripts...
openresty 09:54:39.70 INFO  ==> Initializing OpenResty
realpath: /bitnami/openresty/conf/vhosts: No such file or directory
openresty 09:54:39.70 INFO  ==> ** OpenResty setup finished! **

openresty 09:54:39.72 INFO  ==> ** Starting OpenResty **
2024/08/26 09:54:39 [emerg] 1#1: mkdir() "/opt/bitnami/openresty/nginx/client_body_temp" failed (13: Permission denied)
nginx: [emerg] mkdir() "/opt/bitnami/openresty/nginx/client_body_temp" failed (13: Permission denied)
```

### Additional information

Downgrading to `1.25.3-1` works. Version has been bumped in #71271.

**Comment by juan131** [comment]
Hi @ilpianista 

Thanks so much for reporting this issue, we just released a new container image revision (`1.25.3-2-debian-12-r4`) that should fix it, see:

- https://github.com/bitnami/containers/commit/0b3f20c25a19ad1ad794d53377e5c6fbe3bd824a

Could you give it a try? Thanks in advance.

**Comment by ilpianista** [comment]
@juan131 it does! Thanks!
